test/integration/controller_instance_test.go

```diff
@@ -101,7 +101,6 @@ func TestCreateServiceInstanceNonExistentClusterServiceClassOrPlan(t *testing.T)
 	for _, tc := range cases {
 		tc := tc
 		t.Run(tc.name, func(t *testing.T) {
-			t.Parallel()
```

Comment:
Is this accidental?

Reply:
yes. I was looking at one test with multiple cases to see that individually each test increases in goroutines. Missed it on pushing. Needs to be fixed.